dist/fixto.js

```diff
@@ -329,7 +328,7 @@ var fixto = (function ($, window, document) {
                     }
                 }
             }
-            return top;
+            return top + this.options.top;
```

Comment:
this line is violating the description of the method: "Returns the total outerHeight of the elements passed to mind option. Will return 0 if none."

Maybe we change the description if we want to add the top option.